workflow/rules/svdb.smk

```diff
@@ -33,7 +68,7 @@ rule svdb_query:
     container:
         config.get("svdb_query", {}).get("container", config["default_container"])
     conda:
-        "../envs/svdb_query.yaml"
+        "../envs/svdb.yaml"
     message:
         "{rule}: Use svdb database to filter cnvs in cnv_sv/svdb_query/{wildcards.sample}_{wildcards.type}.normal_filterered.vcf"
```

Comment:
small letters and use {input} or {output} to reduce length of path

Reply:
done